tests/Api/Message/MessageTest.php

```diff
@@ -151,16 +151,7 @@ public function testGetTimestamp()
         $this->assertEquals($datetime->getTimestamp(), $message->getTimestamp());
     }
 
-    public function testGetTtl()
-    {
-        $message = new Message("This is a test message");
-
-        $message->setTtl(3600);
-
-        $this->assertEquals(3600, $message->getTtl());
-    }
-
-    public function testSetTtl()
+    public function testSetAndGetTtl()
     {
         $message = new Message("This is a test message");
```

Comment:
You can assert that the value is, whatever the default value is before the setter call.

After the getter assert you can also set it null again, if it is nullable and assert null

That's how I unit test setters and getters

Reply:
Done